cmd/govim/watcher.go

```diff
@@ -165,6 +87,39 @@ func (m *modWatcher) watch() {
 	}
 }
 
+func eventFilter(root string) func(string) bool {
+	return func(path string) bool {
+		if fi, err := os.Stat(path); err == nil && !fi.IsDir() {
```

Comment:
What happens for _files_ that start with `.` or `_`? Are they notified?

Reply:
Yes. I missed the part about files. Need to update so that the check applies to both directories and files.